pytorch_lightning/trainer/trainer.py

```diff
@@ -761,7 +760,7 @@ def _fit_impl(
         # TODO: ckpt_path only in v2.0
         ckpt_path = ckpt_path or self.resume_from_checkpoint
         self._ckpt_path = self.__set_ckpt_path(
-            ckpt_path, model_provided=model, model_connected=self.lightning_module is not None
```

Comment:
This was a tiny bug, but it's not necessary in the bug-fix because `model` can be used as a truthy check